### PR TITLE
Governance Model Document Creation

### PR DESCRIPTION
**Pull request by ewferg**
* **Tickets addressed:** None
* **Review:** by commit
* **Merge strategy:** Merge (no squash)  

## Description
Initial creation of an Aerie open source governance model. This simply adds a GOVERNANCE.md document to describe our governance.

### CODE REVIEW

**Comment by ewferg** [comment]
There is still some additional things I want to add prior to merging this in (e.g. listing of collaborators, TSC, and PMC), but I wanted to get this out there for an initial review.

File: GOVERNANCE.md
Comment (dyst5422) [inline GOVERNANCE.md]: Typo Aeire

**Comment by camargo** [comment]
This looks good to me. Please run spell check over the document and then I think we can merge.

File: GOVERNANCE.md
Comment (Mythicaeda) [inline GOVERNANCE.md]: Flagging lines that have TBDs so we don't accidentally merge before they're filled in.

File: GOVERNANCE.md
Comment (Mythicaeda) [inline GOVERNANCE.md]: Flagging lines that have TBDs so we don't accidentally merge before they're filled in.

File: GOVERNANCE.md
Comment (Mythicaeda) [inline GOVERNANCE.md]: Two TBDs in this line.

File: GOVERNANCE.md
Comment (ewferg) [reply GOVERNANCE.md]: Fixed!

**Review by Mythicaeda — APPROVED** [review]
Once the TBDs are filled in, this looks good to go!

File: GOVERNANCE.md
Comment (camargo) [reply GOVERNANCE.md]: @ewferg I think we can use [aerie_support@jpl.nasa.gov](mailto:aerie_support@jpl.nasa.gov) for now (same one we use for our [SECURITY.md](https://github.com/NASA-AMMOS/aerie/blob/develop/SECURITY.md) and [CODE_OF_CONDUCT.md](https://github.com/NASA-AMMOS/aerie/blob/develop/CODE_OF_CONDUCT.md) documents)

File: GOVERNANCE.md
Comment (camargo) [reply GOVERNANCE.md]: I think this is okay to merge since we don't actually have these yet. I'll defer to @ewferg tho.

Comment (Mythicaeda) [reply GOVERNANCE.md]: Regarding that deferral, if you agree that it's okay to merge, @ewferg, then my feedback regarding the TBDs changes to a suggestion that the lines with 'TBD' be reworded to flow better, as the phrase `is maintained To Be Determined` was why I thought they were details that needed to be filled in before merging.

Comment (ewferg) [reply GOVERNANCE.md]: I would like to go ahead and knock out the TBDs before we merge this in, hopefully in the next day or two.

File: GOVERNANCE.md
Comment (ewferg) [reply GOVERNANCE.md]: Ok, that sounds good to me for now. Out of curiosity, do you know if we can add non-JPLers to that email list?

File: GOVERNANCE.md
Comment (ewferg) [reply GOVERNANCE.md]: I went ahead and removed the TBDs although the exact PMC members are still in work. I am also debating where to put the PMC list. I am thinking either inline here or in the README. Any preferences?

File: GOVERNANCE.md
Comment (ewferg) [reply GOVERNANCE.md]: Went ahead and added a proposed TSC list.

File: GOVERNANCE.md
Comment (mattdailis) [inline GOVERNANCE.md]:
```suggestion
- Basak Alper Ramaswamy ([bashbash](https://github.com/bashbash)), Jet Propulsion Laboratory

```

File: GOVERNANCE.md
Comment (mattdailis) [inline GOVERNANCE.md:65]: JFYI, aerie-cli is still private

**Review by mattdailis — APPROVED** [review]

File: GOVERNANCE.md
Comment (ewferg) [reply GOVERNANCE.md:65]: I am waiting for the thumbs up from Parker before we go live with this one.

File: GOVERNANCE.md
Comment (ewferg) [reply GOVERNANCE.md]: @camargo, I'll mark this as resolved, but let me know about the email list when you get a chance.

**Pull request merged**
